### ISSUE TITLE
Not working with django-admin-interface

### ISSUE THREAD

**Issue by jgillard** [rajasimon/django-command-palette#2]
Firstly I love the idea of this, thanks for making it!

I followed the install instructions and it didn't immediately work for me. The cause is django-admin-interface, specifically this: https://pypi.org/project/django-admin-interface/#:~:text=Custom%20base%2Dsite.html Is there currently any way to use both of these together? I've poked it a bit already but couldn't work it out.

**Comment by rajasimon** [comment]
Yeah I can see why, this happen because of the styling is no longer applying when use with base.html and right now I've separated the stylesheet and now it's seems working. 

You have to remove the `django.contrib.admin` because `PaletteAdmin` is overriding that so you don't that. 

Please install the latest release  [2022.11.15 ](https://github.com/rajasimon/django-command-palette/releases/tag/2022.11.15)

This is my settings.py when it's work with Django admin interface

```
INSTALLED_APPS = [
    "admin_interface",
    "colorfield",
    "palette.apps.PaletteConfig",
    "palette.apps.PaletteAdminConfig",
    # "django.contrib.admin",
]

```

Let me know if you need further assistance to make it work :)

**Comment by jgillard** [comment]
Thanks for the quick turnaround!

Did you try this with the custom base html? That's the part I don't see how it can work here. As shown in those docs, I currently have a custom admin base that`{% extends "admin_interface:admin/base_site.html" %}`. I can't add a 2nd extend for your template, and your template doesn't extend from this one.

What I've done for now is copy the 3 relevant lines from your base into my base: https://github.com/rajasimon/django-command-palette/blob/main/palette/templates/admin/base.html#L10-L12. This does work!

**Comment by jgillard** [comment]
I mentioned that in my last comment. As I currently extend from `"admin_interface:admin/base_site.html"`, I can't also extend from your template, as you simply can't extend from multiple templates. So the only way to extend from yours is to remove the extend from `admin_interface`, which is the same as removing that dependency.

Regardless I couldn't work out how to extend from yours even if I do remove admin_interface:

`{% extends "palette:admin/base.html" %}` gives me `django.template.exceptions.TemplateDoesNotExist: palette:admin/base.html`
`{% extends "palette/admin/base.html" %}` gives me `django.template.exceptions.TemplateDoesNotExist: palette/admin/base.html`
